providers/okta/okta_auth_test.go

```diff
@@ -40,35 +39,37 @@ func TestWrapError(t *testing.T) {
 	require.Contains(t, oktaErr.Error(), "access denied", "expected the original error text")
 }
 
-// RoundTripperMock implements RoundTripper interface for testing.
-type RoundTripperMock struct {
+type RequestExecutorMock struct {
 	response *http.Response
 	err      error
 }
 
 // RoundTrip returns a pre-defined HTTP response and an error.
-func (o RoundTripperMock) RoundTrip(request *http.Request) (*http.Response, error) {
+func (o RequestExecutorMock) Do(request *http.Request) (*http.Response, error) {
```

Comment:
This was renamed from RoundTrip to Do because we are now using `http.Client` which has a Do instead of RoundTrip.